src/ctim/schemas/weakness.cljc

```diff
@@ -97,7 +99,7 @@
     (f/entry :likelihood v/HighMedLow
              :description (str "how likely the specific consequence is expected "
                                "to be seen relative to the other consequences"))
-    (f/entry :note c/MedString
+    (f/entry :note c/Markdown
```

Comment:
```
(->> "/var/cwe/1000.xml"
       input-stream
       xml-input-stream->weakness-records
       (map :common-consequences)
       (mapcat identity)
       (map :consequence)
       (map :note)
       (remove nil?)
       (map type)
       set) => #{java.lang.String}
```
This change is not needed, there are no cases in the production data where it uses anything other than a simple string.